src/System.Security.Cryptography.Primitives/src/System/Security/Cryptography/HashAlgorithmName.cs

```diff
@@ -51,13 +51,13 @@ namespace System.Security.Cryptography
         /// </summary>
         public static HashAlgorithmName SHA512 { get { return new HashAlgorithmName("SHA512"); } }
 
-        private readonly string _name;
+        private readonly string? _name;
 
         /// <summary>
         /// Gets a <see cref="HashAlgorithmName" /> representing a custom name.
         /// </summary>
         /// <param name="name">The custom hash algorithm name.</param>
-        public HashAlgorithmName(string name)
+        public HashAlgorithmName(string? name)
```

Comment:
Is there any situation where we'd expect you to construct one of these with null?  Regardless, the Name property needs to be nullable, but we could still choose to say that null values aren't intended to be passed to the ctor.  @bartonjs?

Reply:
Since default exists and the guidelines for strongly typed strings explicitly say to not check for null, nullable seems correct.